src/main/scala/scorex/transaction/NgHistoryWriter.scala

```diff
@@ -17,7 +18,7 @@ trait NgHistoryWriter extends HistoryWriter with NgHistory {
 
   def bestLiquidBlock(): Option[Block]
 
-  def forgeBlock(id: BlockId): Option[(Block, DiscardedTransactions)]
+  def forgeBlock(id: BlockId): Option[(Block, Int, DiscardedTransactions)]
```

Comment:
It's better to use `Option[(Block, DiscardedMicroblocks)]`

Reply:
Ok